hack/travis/install-kubernetes-toolchain.sh

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly DESTDIR=$(cd "${DESTDIR:-${HOME}/bin}" && pwd)
```

Comment:
`DESTDIR` is confusing to me. Maybe call it what it is like `BINDIR` or something?

Reply:
It's by analogy to [DESTDIR](https://www.gnu.org/prep/standards/html_node/DESTDIR.html), which was in common usage in  olden times :)

Reply:
No, I'm familiar. This is just an opinion on style.

Reply:
I can change that, but I still need to tag with the official names so I can push them into the Kind cluster.

Reply:
I changed the script to take the target directory as an argument.